src/vs/workbench/contrib/positronPreview/browser/positronPreview.tsx

```diff
@@ -106,7 +107,10 @@ export const PositronPreview = (props: PropsWithChildren<PositronPreviewProps>)
 
 	const urlToolbar = activePreview && activePreview instanceof PreviewUrl;
 	const htmlToolbar = activePreview && activePreview instanceof PreviewHtml;
-	const showToolbar = urlToolbar || htmlToolbar;
+	const basicToolbar =
+		activePreview && activePreview.viewType === 'notebookRenderer';
```

Comment:
Any viewer content coming from the console or notebooks that don't fall under html files or url will have `notebookRenderer` viewType, so it should not ever be the case that multiple toolbars show.